mne/io/base.py

```diff
@@ -218,8 +218,7 @@ class _BaseRaw(ProjMixin, ContainsMixin, UpdateChannelsMixin,
 
     Subclasses must provide the following methods:
 
-        * _read_segment_file(self, data, idx, offset, fi, start, stop,
-                             cals, mult)
+        * _read_segment_file(self, data, idx, fi, start, stop, cals, mult)
```

Comment:
Simplified `_read_segment_file`. Note that `stop` is now *exclusive* like in all other Python contexts, instead of being *inclusive* as it was before.